kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/ServiceControlCli.scala

```diff
@@ -17,33 +17,12 @@
 
 package org.apache.kyuubi.ctl
 
-import scala.collection.mutable.ListBuffer
-
 import org.apache.kyuubi.Logging
-import org.apache.kyuubi.config.KyuubiConf.{ENGINE_SHARE_LEVEL, ENGINE_SHARE_LEVEL_SUBDOMAIN, ENGINE_TYPE}
-import org.apache.kyuubi.ha.HighAvailabilityConf._
-import org.apache.kyuubi.ha.client.{DiscoveryClientProvider, ServiceNodeInfo}
-import org.apache.kyuubi.ha.client.DiscoveryClient
-import org.apache.kyuubi.ha.client.DiscoveryPaths
-
-private[ctl] object ServiceControlAction extends Enumeration {
-  type ServiceControlAction = Value
-  val CREATE, GET, DELETE, LIST = Value
-}
-
-private[ctl] object ServiceControlObject extends Enumeration {
-  type ServiceControlObject = Value
-  val SERVER, ENGINE = Value
-}
 
 /**
  * Main gateway of launching a Kyuubi Ctl action.
  */
 private[kyuubi] class ServiceControlCli extends Logging {
```

Comment:
should we rename ServiceControlCli => ControlCli in this PR?
Because it will not be only for service control.

Reply:
@ulysses-you  @pan3793

Reply:
ServiceControlAction => ControlAction
ServiceControlObject => ControlObject
ServiceControlCli => ControlCli
ServiceControlCliArguments => ControlCliArguments
ServiceControlCliArgumentsParser => ControlCliArgumentsParser